src-options/index.html

```diff
@@ -75,7 +76,7 @@ <h1>Stutter Configuration</h1>
 			</label>
 
 			<p class="keybind-label">Keybindings</p>
-			<p>To change a binding click on a row below, press the key combination you prefer, then save.</p>
+			<p>To change a binding click on a row below, press the key combination you prefer, then <button class="submit" type="submit">Save</button>.</p>
```

Comment:
Added the "Save" button here too cos it scrolls off the screen and I forget to click it to make the changes stick.